server/utils/generateFileSystemSafeName.ts

```diff
@@ -5,12 +5,10 @@
  * @param {String} string
  * @param {String} replacer (optional) character to replace invalid characters
  */
-function generateFileSystemSafeName(string, replacer) {
+export function generateFileSystemSafeName(string: string, replacer: string) {
```

Comment:
is there any way we can rename this from string to something else? i assume it's not causing a bug but it just makes me nervous haha